clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/security/AmazonCredentials.java

```diff
@@ -49,6 +49,7 @@
     private final List<AWSRegion> regions;
     private final List<String> defaultSecurityGroups;
     private final List<LifecycleHook> lifecycleHooks;
+    private final Boolean allowPrivateThirdPartyImages;
```

Comment:
can you make this little b boolean and assign a default value in the constructor? If we want to use this in non-groovy code it would be nice to not have to null check it